extension/src/extension.ts

```diff
@@ -6,7 +6,8 @@ import { URL as _URL, URLSearchParams as _URLSearchParams } from 'whatwg-url'
 Object.assign(_URL, self.URL)
 Object.assign(self, { URL: _URL, URLSearchParams: _URLSearchParams })
 
-import { activateBasicCodeIntel, registerFeedbackButton } from '@sourcegraph/basic-code-intel'
+import { initLSIF } from '@sourcegraph/basic-code-intel'
+import { mkIsLSIFAvailable } from '@sourcegraph/basic-code-intel/lib/lsif'
```

Comment:
(not blocking) Could the package export this from the index so it is public API?

Reply:
Yup, submitted https://github.com/sourcegraph/sourcegraph-basic-code-intel/pull/133